.travis.yml

```diff
@@ -33,7 +31,7 @@ install:
   - make
   - ./webdis &
   - popd
-  - travis_retry pip install . --process-dependency-links
+  - travis_retry pip install . --no-binary redbiom
```

Comment:
As I understand it, this forces redbiom to be installed from source, rather than precompiled binary? I take it this is to resolve one or more dependency issues?

Reply:
Well, turns out that the wheel redbiom binary is broken and the quickest way forward is to install from source ...

Reply:
sounds good then. Compilation for redbiom itself won't add appreciable compilation time. If versioning is 'latest' that could introduce variability over time, but no more so than using the latest binary.